.github/ISSUE_TEMPLATE/give_a_talk.yml

```diff
@@ -1,6 +1,6 @@
-name: "\U0001F4E3 Give a TTW talk"
```

Comment:
I think we can keep the TTW, after all not collecting all kinds of talks here :)

Reply:
@JimMadge made this change [here](https://github.com/the-turing-way/the-turing-way/commit/bb8c9ead733f745a4ad62a31b60e1c30860ad2b5#diff-95db8faa6474b3738cefcf1f6832722dbb62b38f90d3cd59ebba8940ff88b5fcR1) - what're your thoughts @JimMadge?

Reply:
I don't think it hurts to keep. Makes it clear what kinds of talks we mean.

Might change TTW to _The Turing Way_.